ml-agents-envs/mlagents_envs/rpc_communicator.py

```diff
@@ -76,7 +75,8 @@ def check_port(self, port):
         Attempts to bind to the requested communicator port, checking if it is already in use.
         """
         s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        if platform == "linux" or platform == "linux2":
+            s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
Could you explain this change?  A comment might actually be helpful for clarity anyway.

Reply:
Will do